.circleci/config.yml

```diff
@@ -228,6 +235,9 @@ jobs:
     resource_class: medium
     steps:
       - custom_attach_workspace
+      - install_python
+      - setup_bazel_rbe
+      - run: sudo cp .circleci/bazel.rc /etc/bazel.bazelrc
```

Comment:
Can this step just be added to the `setup_bazel_rbe` command?  This would prevent us having to repeat it in all of the spots using rbe